network/writer.go

```diff
@@ -25,8 +26,13 @@ type DefaultErrorJson struct {
 func WriteJson(w http.ResponseWriter, v any) {
 	w.Header().Set("Content-Type", jsonMediaType)
 	w.WriteHeader(http.StatusOK)
+	if err := apimiddleware.ProcessMiddlewareResponseFields(v); err != nil {
```

Comment:
This package should not rely on API middleware. In fact, we want to remove the middleware entirely in the future.